feat(admin-panel): Add information pertaining to bounceType - WIP

xlisachan opened the pull request:
Because:

* it would provide more information when clearing email bounces

This commit:

- [x] adds descriptions for bounce type/subtype based on Amazon SES
- [x] updates/renames `ResultListItem` to `ResultTableRow` 
- [x] fixes the `verified`/`not verified` style as they were not appearing green/red
- [x] updates Connected Services as tables as well
- [ ] adds tests

Closes #12155

To do:
- [ ] PR adding descriptions to documentation in Ecosystem Platform

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
BEFORE
<img width="1287" alt="Screen Shot 2022-05-13 at 2 50 51 PM" src="https://user-images.githubusercontent.com/28129806/168348593-73d36404-bd3c-4571-83f8-b278f62bbf42.png">

AFTER
<img width="1286" alt="Screen Shot 2022-05-13 at 2 13 33 PM" src="https://user-images.githubusercontent.com/28129806/168347631-ec19b38b-10bb-487a-8aef-352cf9378fc8.png">

Connected Services
<img width="1309" alt="Screen Shot 2022-05-13 at 2 53 28 PM" src="https://user-images.githubusercontent.com/28129806/168353607-ede75e70-3bb1-490f-9b44-5b5e18141847.png">

xlisachan commented:
We use `dedent` in the Swagger docs to help with multi-line strings and markdowns, but it doesn't seem like they're being applied. I tried using `ts-dedent` since these are .ts files, but that doesn't seem to work either. Any ideas on what could be happening? 

- There is supposed to be a new line before and after "⚠️ Important"
<img width="999" alt="Screen Shot 2022-05-13 at 2 10 45 PM" src="https://user-images.githubusercontent.com/28129806/168373837-bfce2413-11ed-46ee-94cd-4868a5f10fd6.png">

- The text is not a link
<img width="1047" alt="Screen Shot 2022-05-13 at 2 08 41 PM" src="https://user-images.githubusercontent.com/28129806/168373858-3aa11943-6140-411c-a354-4dba53f21de6.png">

dschom commented:
@xlisachan The dedent library just smartly strips leading white space on newlines, but it doesn't actually add any html, which is why you aren’t seeing any effect. There are two ways I can think of dealing with this. One approach could be using wrapping the text with <pre> block and applying the appropriate styling. Another option, that I think would be better, is to return jsx with html tags to render the output as desired from the bounce-descritions.tsx.

dschom reviewed (approved):
Aside from the dedent issue, which isn't that big a deal imo, this looks good to me.

xlisachan commented:
Thanks for the feedback, @dschom ! I have made the following updates:
- Removed `dedent`/`ts-dedent` as it was not working as expected
- Updated the descriptions into an array and used `.map()` to inject each string into a `p` tag instead, per @LZoog's suggestion
- Updated the first screenshot within the PR issue
- Updated the description file from a `.tsx` to `.ts` file, per @LZoog's suggestion
- Added `Undetermined` to `BounceType`, and `OnAccountSuppressedList` to `BounceSubType`

Can you please review this PR again when you have a moment? Thank you!

xlisachan commented:
Noting that Dan reviewed the revisions and gave the OK :)
<img width="626" alt="Screen Shot 2022-05-17 at 11 59 29 AM" src="https://user-images.githubusercontent.com/28129806/168856258-dcd9c250-d7ee-4a97-aa76-729b1481ce3a.png">
